Source/WebInspectorUI/UserInterface/Views/ColorSwatchDisplay.js

```diff
@@ -0,0 +1,60 @@
+WI.ColorSwatchDisplay = class ColorSwatchDisplay extends WI.Object
+{
+    constructor(delegate, colorPalettes)
+    {
+        super();
+
+        this.delegate = delegate;
+
+        this.element = document.createElement("div");
+        this.element.classList.add("swatch-display");
+
+        this._title = this.element.appendChild(document.createElement("p"));
+        this._title.innerText = "styles";
+
+        this._swatchWrapper = this.element.appendChild(document.createElement("div"));
```

Comment:
Semantically in HTML, the swatch wrapper is a list not a generic `div`. It will make more sense for assistive technologies as well if this is a `ul` or `ol` (though the ordering is ambiguous) with list items (`li`) containing the swatches themselves.